### PR TITLE
Add init seeding to token embedder.

### LINKED ISSUE #35: Identify and fix source of randomness in transformer initialization
Currently, if you run `animated-transformer/src/lib/trainer/basic_transformer_trainer.spec.ts` two times in a roll, the initial loss does not match. (before the sgd step).

The task seems to be generating reproducible results, and the neural network initialized parameters are also reproducible. However, once you initialize the train state, the loss is different on every run.

### PR DESCRIPTION
**Pull request by aliciafmachado**
Fix #35.

### CODE REVIEW

**Comment by iislucas** [comment]
thanks! LGTM.

**Review by iislucas — APPROVED** [review]
LGTM
